docs/REST.md

```diff
@@ -780,66 +779,24 @@ However, `Future` is low level and limited in many ways (e.g. there is no way to
 asynchronous computation starts). It is possible to use other task-like containers, e.g.
 [Monix Task](https://monix.io/docs/2x/eval/task.html) or [Cats IO](https://typelevel.org/cats-effect/).
 
-In order to do that, you must provide some additional "serialization" implicits which will translate your
-wrapped results into `Async[RestResponse]`. Just like when
-[providing serialization for third party type](#providing-serialization-for-third-party-type),
-you should put that implicit into a trait and inject it into REST API trait's companion object.
+In order to do that, you must provide some additional "serialization" implicits which will make the macro engine
+understand how to translate between `Async[T]` and `Task[T]` for arbitrary type `T`. This is controlled by
```

Comment:
Is it translation or conversion?

Reply:
What's the difference? For me, "conversion" implies actual _data_ conversion. In this case it's more like wrapping or adapting, hence "translation".

Reply:
Exactly my thoughts. I just wanted to make sure it's supposed to rather be a wrapper than a converter.